tests/acceptance/features/cliDataExporter/import.feature

```diff
@@ -21,3 +21,16 @@ Feature: An administrator wants to import a user using the commandline
     And as "usertrash" file "AFolder/DeletedFile.txt" should exist
     And the content of file "AFolder/DeletedFolder/fileinfolder.txt" for user "usertrash" should be "text in file in deleted folder"
     And the content of file "AFolder/DeletedFile.txt" for user "usertrash" should be "text in deleted file"
+
+  Scenario: Restore and rename files in trashbin after import
+    Given a user has been imported from path "trashbinExport/usertrash" using the occ command
+    And the administrator has changed the password of user "usertrash" to "123456"
+    And user "usertrash" has restored the folder with original path "AFolder/DeletedFolder"
+    And user "usertrash" has restored the file with original path "AFolder/DeletedFile.txt"
```

Comment:
IMHO according to your `scenario` description this should be `when` step!

Reply:
Done